### PR TITLE
React upgrade

### PR DESCRIPTION
**Pull request by hqazi99**

### CODE REVIEW

**Review by oliverjohnstone — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -13,17 +13,18 @@
     "build": "gulp clean && NODE_ENV=production gulp build"
   },
   "peerDependencies": {
-    "react": "^15.4.2",
-    "react-dom": "^15.4.2"
+    "react": "16.0.0",
```
Comment (lainaireau) [inline package.json]: prob want ^15.4.2 || ^16.0.0 here

File: package.json [hunk 2]
```diff
@@ -13,17 +13,18 @@
...
   },
   "devDependencies": {
     "babel-eslint": "^4.1.3",
     "eslint": "^1.6.0",
     "eslint-plugin-react": "^3.5.1",
     "gulp": "^3.9.0",
-    "react": "^0.14.0",
     "react-component-gulp-tasks": "^0.7.6",
-    "react-dom": "^0.14.0"
+    "react": "16.0.0",
```
Comment (lainaireau) [inline package.json]: don't think we need it on devdeps here

File: package.json [hunk 3]
```diff
@@ -13,17 +13,18 @@
     "build": "gulp clean && NODE_ENV=production gulp build"
   },
   "peerDependencies": {
-    "react": "^15.4.2",
-    "react-dom": "^15.4.2"
+    "react": "16.0.0",		
+    "react-dom": "16.0.0",
+    "prop-types": "^15.5.10"
```
Comment (lainaireau) [inline package.json]: proptypes should be at ^15.6.0 https://www.npmjs.com/package/prop-types

**Review by lainaireau — CHANGES_REQUESTED** [review]
minor stuffs :-)

**Review by lainaireau — APPROVED** [review]
👍

**Pull request merged**